lib/config_builder/model/vm.rb

```diff
@@ -166,8 +172,9 @@ def initialize
   end
 
   def to_proc
+    autostart = attr(:autostart).nil? ? true : attr(:autostart)
```

Comment:
I believe this can be done by adding `:autostart => true` in the `#initialize` method for the `@defaults` hash instead of dealing with defaults in the body of this method.